packages/dashboard/src/customization/propertiesSections/lineAndScatterStyleSettings/yAxis.tsx

```diff
@@ -44,34 +45,38 @@ export const YAxisSection: FC<YAxisSectionOptions> = ({ visible, min, max, setVi
 
   return (
     <ExpandableSection
+      className='accordian-header accordian-custom-header'
       expanded={expanded}
       onChange={(e) => setExpanded(e.detail.expanded)}
       headerText={
-        <SpaceBetween size='s' direction='horizontal'>
+        <div className='yaxis-header'>
```

Comment:
I think this removes the padding between the controls that were being applied by the `size='s'` Can we check with UX that this is desired?

Reply:
This change won't affect the padding thing as the expected UX is to align "View on chart toggle" to right.

![image](https://github.com/awslabs/iot-app-kit/assets/81667589/d2333b05-73e3-48df-912c-634c58360757)